src/CSSBattle/1_SimplySquare/tin.html

```diff
@@ -0,0 +1,13 @@
+<div></div>
```

Comment:
could not see battle #1 and #2, file named incorrectly

Reply:
commit 6188481e167a19214605c9ad16a30a54c7a86507